src/ucp/core/ucp_listener.c

```diff
@@ -54,7 +75,12 @@ static ssize_t ucp_listener_conn_request_callback(void *arg, const void *conn_pr
                                           &prog_id);
     }
 
-    return 0;
+    return 0; /* replying to client by wireup request instead of private data */
```

Comment:
following the removal of the reply cb, will need to return UCS_OK here.

Reply:
yes, and also remove the reply argument and not pass callback on client side..